### DISCUSSION TITLE
JIRA Search integration for bug metrics

### DISCUSSION THREAD

**Discussion by jtaleric** [cloud-bulldozer/ocp-perf-dashboard#78] [Ideas]
We are being asked to add bug metric data to our CPT Dashboard. Since we incorporate multiple products, we a generic way to capture the bugs. 

Data we are looking to capture : 
- Number of bugs open in a given date range.
- State of each bug 
- Time bug has been open 

We have discussed two options : 
1. Each Product owner adding to the CPT Dashboard will provide a JIRA Query / JQL that provides the necessary information we are looking for *see Data we are looking to capture*. 
2. Each Product owner adding to the CPT Dashboard will provide the products to query from with the corresponding label or component for the CPT Dashboard backend code to query.

**Comment by chentex** [comment]
In my case I think option 2 is what I feel the most comfortable, but this might lead to investing in a lot of testing the parameters, to see if they return what we expect.
